pkg/driver/controller.go

```diff
@@ -234,7 +247,25 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	rootDirName := volName
-	rootDir := basePath + "/" + rootDirName
+	// Check if a custom structure should be imposed on the access point directory
+	if value, ok := volumeParams[SubPathPattern]; ok {
+		if len(value) == 0 {
+			return nil, status.Errorf(codes.InvalidArgument, "No pattern specified for subPath.")
```

Comment:
Nit: this should say `No pattern specified for subPathPattern.` instead of `No pattern specified for subPath.`.

Reply:
Resolved this by removing the check completely to support the use case referred to above.